REPOSITORIES.md

```diff
@@ -0,0 +1,179 @@
+# Repositories
+
+This document lists all repositories, sorted by their scope, within [Cilium](https://github.com/cilium).
+
+You can read more about the scope and lifecycle of repositories in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
+
+**Table of Contents**
+
+* [```Core``` Scope](#core-scope)
+* [```Ecosystem``` Scope](#ecosystem-scope)
+* [```Infra``` Scope](#infra-scope)
+* [```Special``` Scope](#special-scope)
+* [```Historical``` Scope](#historical-scope)
+
+## ```Core``` Scope
+
+Learn more about the ```Core``` scope in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORY-GUIDELINES.md#core).
+
+|   Name  | Description |
+|:--------|:------------|
+| [cilium/charts](https://github.com/cilium/charts) | Helm charts repository for the Cilium project |
+| [cilium/cilium](https://github.com/cilium/cilium) | eBPF-based Networking, Security, and Observability |
+| [cilium/cilium-cli](https://github.com/cilium/cilium-cli) | CLI to install, manage & troubleshoot Kubernetes clusters running Cilium |
+| [cilium/cilium.io](https://github.com/cilium/cilium.io) | Cilium project website |
+| [cilium/ebpf](https://github.com/cilium/ebpf) | A Go library to read, modify and load eBPF programs and attach them to various hooks in the Linux kernel |
+| [cilium/hubble](https://github.com/cilium/hubble) | Network, Service & Security Observability for Kubernetes using eBPF |
+| [cilium/hubble-ui](https://github.com/cilium/hubble-ui) | User interface for [cilium/hubble](https://github.com/cilium/hubble)
+| [cilium/pwru](https://github.com/cilium/pwru) | eBPF-based Linux kernel networking debugger |
+| [cilium/tetragon](https://github.com/cilium/tetragon) | eBPF-based Security Observability and Runtime Enforcement |
+
+## ```Ecosystem``` Scope
+
+Learn more about the ```Ecosystem``` scope in [REPOSITORY-GUIDELINES.md](https://github.com/cilium/community/blob/main/REPOSITORY-GUIDELINES.md#ecosystem).
+
+|   Name  | Description |
+|:--------|:------------|
+| [cilium/arping](https://github.com/cilium/arping) | Go Library to ping a host per arp datagram or query a host mac address
```

Comment:
This looks like it's a fork, do we want to classify forks under Ecosystem?

Reply:
Moved to Special!